src/components/codeSpace/Resizable.tsx

```diff
@@ -7,9 +7,9 @@ import ExerciseInfoBox from "./exerciseInfoBox";
 import { api } from "@/trpc/server";
 import CodeEditor from "../ui/CodeEditor";
 
-import ScrollAreaDemo from "./testsuite";
+import TestSuiteScrollArea from "./Testsuite";
```

Comment:
You import it as TestSuiteScrollArea, but the exported component is "testSuite" in the codeSpace/testsuite.tsx file. This is inconsistent